### ISSUE TITLE
formatter: oxfmt 0.57.0 SCSS comments stripped regressions

### ISSUE THREAD

**Issue by KieranP** [oxc-project/oxc#24193]
Latest oxfmt release switched CSS parser, has issue with SCSS style comments (SCSS allows comment using `//` which are stipped from final output, compared with `/* */` which is preserved in final output.

### Input

```scss
@use 'bootstrap/scss/bootstrap.scss' with (
  // Colors
  $primary: variables.$primary,
  $secondary: variables.$secondary,
  $success: variables.$success,
  $warning: variables.$warning,
  $danger: variables.$danger,
  $info: variables.$info,
  $dark: variables.$dark,
  $light: variables.$light,

  // Components
  $box-shadow: variables.$box-shadow,

  // Spacing
  $spacer: variables.$spacer,

  // Tooltips
  $tooltip-font-size: variables.$tooltip-font-size
);
```

### Config

```jsonc
{}
```

### Oxfmt output

Oxfmt version: `0.57.0`
```scss
@use 'bootstrap/scss/bootstrap.scss' with (
  $primary: variables.$primary,
  $secondary: variables.$secondary,
  $success: variables.$success,
  $warning: variables.$warning,
  $danger: variables.$danger,
  $info: variables.$info,
  $dark: variables.$dark,
  $light: variables.$light,
  $box-shadow: variables.$box-shadow,
  $spacer: variables.$spacer,
  $tooltip-font-size: variables.$tooltip-font-size
);
```

(comments stripped)

### Prettier output

Comments preserved:

```scss
@use 'bootstrap/scss/bootstrap.scss' with (
  // Colors
  $primary: variables.$primary,
  $secondary: variables.$secondary,
  $success: variables.$success,
  $warning: variables.$warning,
  $danger: variables.$danger,
  $info: variables.$info,
  $dark: variables.$dark,
  $light: variables.$light,

  // Components
  $box-shadow: variables.$box-shadow,

  // Spacing
  $spacer: variables.$spacer,

  // Tooltips
  $tooltip-font-size: variables.$tooltip-font-size
);
```

**Comment by KieranP** [comment]
Looks like it's not just SCSS comments, but also normal CSS comments. This too has it's comments stripped after formatting:

```scss
@use 'bootstrap/scss/bootstrap.scss' with (
  /* Colors */
  $primary: variables.$primary,
  $secondary: variables.$secondary,
  $success: variables.$success,
  $warning: variables.$warning,
  $danger: variables.$danger,
  $info: variables.$info,
  $dark: variables.$dark,
  $light: variables.$light,

  /* Components */
  $box-shadow: variables.$box-shadow,

  /* Spacing */
  $spacer: variables.$spacer,

  /* Tooltips */
  $tooltip-font-size: variables.$tooltip-font-size
);
```

**Comment by leaysgur** [comment]
Thanks for the report!

This will be fixed by #24205, will be released as v0.58.
